text/0000-deprecation-ember-string.md

```diff
@@ -0,0 +1,62 @@
+- Start Date: 2017-07-14
+- RFC PR: (leave this empty)
+- Ember Issue: (leave this empty)
+
+# Summary
+
+This PR proposes the deprecation of `Ember.String`. The following methods will be moved to an addon:
+
+- `camelize`
+- `capitalize`
+- `classify`
+- `dasherize`
+- `decamelize`
+- `fmt`
+- `loc`
+- `underscore`
+- `w`
+
+The remaining methods (`htmlSafe` and `isHTMLSafe`) will be moved to `@ember/component`.
+
+In both cases, the String prototype won't be extended anymore.
```

Comment:
That's a pretty big breaking change. I know it's not great to extend prototypes but I know that at least our app pretty heavily relies on this. It would take quite some effort to audit and fix all the uses.

Perhaps this could be changed to:

> In both cases, the String prototype won't be extended anymore *by default*.